packages/lit-element/src/lib/platform-support.ts

```diff
@@ -367,48 +348,52 @@ let currentScope: string;
       if (needsStyleScoping && renderingIntoShadowRoot) {
```

Comment:
`renderingIntoShadowRoot` is `true` here due to test on L330 right?

Reply:
Fixed.